tests/suite/index.js

```diff
@@ -16,6 +16,13 @@ describe('/', () => {
         expect(response).to.have.header('Cache-Control', 'public, max-age=30672000, immutable'); // 355 days
         done();
     });
+    it('returns the correct CORS and Cache headers for OPTIONS', done => {
+        request().options('/').redirects(0).end((err, res) => {
+            expect(res).to.have.header('Access-Control-Allow-Origin', '*');
+            expect(res).to.have.header('Cache-Control', 'public, max-age=30672000, immutable'); // 355 days
+            done();
+        });
+    });
```

Comment:
```suggestion
    it('returns the correct CORS headers for OPTIONS', done => {
        request().options('/').redirects(0).end((err, res) => {
            expect(res).to.have.header('Access-Control-Allow-Origin', '*');
            done();
        });
    });
```

This currently fails when tests are run, I'm guessing custom headers aren't mirrored from the GET routes, which is fine.

Reply:
Oh yeah that makes sense. We can add it, but we `Cache-Control` doesn't really make sense for `options`

Reply:
agreed